cmd/api-server/main.go

```diff
@@ -93,16 +94,35 @@ func main() {
 
 	resultsRepository := result.NewMongoRespository(db)
 	testResultsRepository := testresult.NewMongoRespository(db)
-	configRepository := config.NewMongoRespository(db)
+	configRepository := configmongo.NewMongoRespository(db)
+	configMapConfig, err := configmap.NewConfigMapConfig(os.Getenv("APISERVER_CONFIG"), namespace)
+	ui.ExitOnError("Getting config map config", err)
 
+	ctx := context.Background()
 	// try to load from mongo based config first
-	telemetryEnabled, err := configRepository.GetTelemetryEnabled(context.Background())
+	telemetryEnabled, err := configMapConfig.GetTelemetryEnabled(ctx)
 	if err != nil {
 		// fallback to envs in case of failure (no record yet, or other error)
 		telemetryEnabled = envs.IsTrue("TESTKUBE_ANALYTICS_ENABLED")
 	}
 
-	clusterId, err := configRepository.GetUniqueClusterId(context.Background())
+	var clusterId string
+	config, err := configMapConfig.Get(ctx)
```

Comment:
We still have GetUniqueClusterId method? Why not use it here?

Reply:
Ok I see here is just writing to storage in case

Reply:
yes, we try to read from config map, if we can't get it, then we try to read mongo config, and if we don't have access to mongo config, we generate and save a new  cluster id